Makefile

```diff
@@ -30,3 +26,23 @@ test :
 cover :
 	go test ./msgservice ./dbservice ./router ./formatting ./data ./regoservice ./routes -v -coverprofile=cover.out
 	go tool cover -html=cover.out
+
+develop :
+	@echo "Running Postee UI...."
+	docker-compose up
+
+docker-webhook :
+	@echo "Building image...."
+	docker build --no-cache -t aquasec/postee:webhook -f Dockerfile .
+	docker run -p 8082:8082 -p 8445:8445 aquasec/postee:webhook
+
+docker-release : build
```

Comment:
better naming could used here:
- `develop` - it uses docker compose script. But this mode not linked with development. It can be useful for quick testing or as example for k8s config
- `docker-webhook` is used to build docker image from repository directly.
- `docker-release` it uses dockerfile intended for `goreleaser`. Also goreleaser uses `Dockerfile.release` directly. So I'm not sure if it should be part of `Makefile`